livekit-plugins/livekit-plugins-google/livekit/plugins/google/stt.py

```diff
@@ -303,10 +303,12 @@ async def input_generator():
                 retry_count = 0  # connection successful, reset retry count
 
                 await self._run_stream(stream)
-            except Aborted:
-                logger.error("google stt connection aborted")
-                break
-            except Exception as e:
+            except (Aborted, Exception) as e:
+                logger.error(
```

Comment:
We're already logging bellow. Is the Aborted error only fired when the maximum stream duration is reached? If so, we shouldn't log the error to make it more seamless to the user

Reply:
I think that's a diffrent issue, @davidzhao answered here https://github.com/livekit/agents/issues/852#issuecomment-2463792401
Aborted will fired when someone closes the agent session or some other concurrency issues 
![image](https://github.com/user-attachments/assets/9b8411bf-e2a1-4756-8f5f-506b6757cf54)
https://cloud.google.com/speech-to-text/v2/docs/reference/rest/v2/Code